packages/dck-redux/tslint.json

```diff
@@ -5,7 +5,8 @@
   "rules": {
     "ordered-imports": false,
     "object-literal-sort-keys": false,
-    "interface-over-type-literal": false
+    "interface-over-type-literal": false,
+    "no-console": false
```

Comment:
please don't add this ignore rule. We are providing a lib, we should not be using console